Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -115,7 +115,7 @@ abstract class FunctionalTestCase extends BaseTestCase
      * A default list of core extensions is always loaded.
      *
      * @see FunctionalTestCaseUtility $defaultActivatedCoreExtensions
-     * @var array
+     * @var string[]
```

Comment:
We can be even more specific:

```suggestion
     * @var array<int, string>
```

Reply:
I've always thought this is exactly the same but you are right, the first one means `array<string|int, string>` see https://github.com/phpDocumentor/TypeResolver/blob/1.x/src/Types/AbstractList.php#L44.

Reply:
… or `array<array-key, string>` (which semantically is the same, but communicates another intent).

Reply:
care to update to Olivers first remark?